modules/component/src/componentNext.ts

```diff
@@ -39,6 +39,12 @@ type LActionTypes<A> = A extends Action<any, infer T> ? T : never
 type LActionValues<A> = A extends Action<infer V, any> ? V : never
 type LObjectValues<O> = O extends {[k: string]: infer S} ? S : unknown
 type LActionValueForType<A, T> = A extends Action<infer V, T> ? V : never
```

Comment:
we can make 

```
LActionValueForType<A, T, D = never> = A extends Action<infer V, T> ? V : D
```

Reply:
this will not work when we pass `A` as  Action set.